src/contribute/release-guide.md

```diff
@@ -398,6 +398,40 @@ Staging binaries
 
 Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
+
+### Build python wheels and upload it to test.pypi.org
```

Comment:
We should put some TODOs in here to make it clear this is no the long-term process. 

Probably we should just have a single repo apache/beam-wheels (which all committers can push to) and upload to somewhere other than test.pypi.org.

Reply:
I agree add TODOs here, but I'm not sure whether it's OK to have TODO in public doc.